### ISSUE TITLE
Release v0.1.0 on PyPI

### ISSUE THREAD

**Issue by namurphy** [wtbarnes/fiasco#174]
There is some really great functionality in here, and it would be worth it to do an initial release  (i.e. `v0.1.0b1` or `0.1.0`) to PyPI.  For an initial release, it will be generally understood that things are unstable or not feature complete, and I think it's ready for that.

Thank you!

**Comment by wtbarnes** [comment]
Thanks for the feedback! It's good to know that there is outside interest in using fiasco.

I'm happy to get a v0.1 release out the door soon. The one thing I would still like to wrap up before that is #42 as that is an actual bug  (and not just a lack of a feature!). I agree that everything else can just be deferred to a feature to be implemented in later versions.

Given this, I've amended the milestones to add this single issue to 0.1. If there's not a 0.1 release by that milestone deadline, please ping me again!

**Comment by namurphy** [comment]
Awesome, that's great to hear!  If you'd like, I'd be happy to do a co-working session over Zoom for an hour or two in the next few weeks.  I could help out with code reviews and some package infrastructure tasks.

**Comment by wtbarnes** [comment]
That'd be great! My plan is to devote the entire three days before the Thanksgiving holiday to fiasco work so if you have some time during those days, that would work well.

**Comment by namurphy** [comment]
Cool!  I don't have anything else scheduled on Monday, November 21 so I'd be happy to set aside a few hours, and perhaps even the whole day.  In the meantime, I'll give some thought on package infrastructure tasks that I could work on, and might raise a few more issues too.

**Comment by namurphy** [comment]
@wtbarnes — Are there any issues that still need to be addressed for 0.1.0 besides #42, #178, and #181?

**Comment by wtbarnes** [comment]
Nope that should be it. I'm going to look at #42 again on Wednesday and if I can't find a satisfactory solution by the end of the week, I'll probably just remove that function for now and cut a release. The other remaining issues are trivial.

**Comment by namurphy** [comment]
Awesome!  Feel free to ping me for a code review if it'd help.  Looking forward to the release!

**Comment by wtbarnes** [comment]
One potential roadblock here is that the name "fiasco" has already been taken on PyPI: https://pypi.org/project/fiasco/ (wishing I had taken it when I started this project!). The project however appears to be dead. I've contacted the maintainers to see if they would be willing to transfer the project to me so we'll see what they say.

[PEP 541](https://peps.python.org/pep-0541/) also provides some good information on these kinds of situations.
